src/network/networkd-link.c

```diff
@@ -70,6 +70,9 @@ static bool link_dhcp6_enabled(Link *link) {
         if (!link->network)
                 return false;
 
+        if (streq_ptr(link->kind, "vrf"))
+                return false;
```

Comment:
Why we want to disable on vrf slave ?

Reply:
Oh, dropped.

Not sure, but `LinkLocalAddressing=` does not work on vrf devices. So, I thought vrf devices do not support addressing. But, it is wrong, hence, dropped. Thank you.

Reply:
No it's correct. Please include that .  I also didn't knew that they don't have LL addresses.
https://elixir.bootlin.com/linux/latest/source/drivers/net/vrf.c#L1089